cli.go

```diff
@@ -1,5 +1,87 @@
 package main
 
-func main() {
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+// Command struct is a representation of an isolated command executed by a user
+type Command struct {
+	Command string `json:"Command"`
+	Username string `json:"Username"`
+	Amount string `json:"Amount"`
```

Comment:
the amount is deliberately specified as a string. There's a case in the user workload file where a user command doesn't specify the amount intentionally. In this case, we shouldn't automatically set the amount to 0. An error response would be more appropriate. Another workaround would be setting the amount to nil, if the amount is not specified. But Golang doesn't allow setting a nil value to type int vars.

Reply:
Ok. But is there a reason why you specified `json` in the `struct`? Is this for the http requests?

Reply:
Yes, these tags help in the conversion of golang structs to json data.  Then the json data can be sent over the wire. 

See this [blog](https://www.golangprograms.com/how-to-convert-go-struct-to-json.html)